app/controllers/courses_controller.rb

```diff
@@ -1,16 +1,25 @@
 class CoursesController < InheritedResources::Base
   load_and_authorize_resource
 
+  def create
+    @course = Course.new(course_params)
+    if @course.save
+      redirect_to courses_path, notice: 'Course saved.'
+    else
+      render :new
+    end
+  end
+
   def update
     if params[:commit] == 'Save order'
       update_section_and_lesson_order
       redirect_to courses_path, notice: "Order updated."
     else
-      course = Course.find(params[:id])
-      if course.update(course_params)
-        redirect_to course_path(course), notice: "Course updated."
+      @course = Course.find(params[:id])
+      if @course.update(course_params)
+        redirect_to course_path(@course), notice: "Course updated."
       else
-        render 'edit'
+        render :edit
```

Comment:
This is somewhat just personal preference, but I lean towards using strings for template names, so that you can specify the path of another folder if needed (eg `render 'sections/edit'`).